logger/fluentd/Dockerfile

```diff
@@ -1,70 +1,19 @@
-# Logging API. This configuration assumes that the host performning
-# the collection is a VM that has been created with a logging.write
-# scope and that the Logging API has been enabled for the project
-# in the Google Developer Console.
-
-FROM gcr.io/google_containers/ubuntu-slim:0.6
-
-
-# Disable prompts from apt
-ENV DEBIAN_FRONTEND noninteractive
-
-# Install build tools
-RUN apt-get -qq update && \
-    apt-get install -y -qq curl ca-certificates gcc g++ make bash sudo && \
-    apt-get install -y -qq --reinstall lsb-base lsb-release && \
-    # Install logging agent and required gems
-    /usr/bin/curl -sSL https://toolbelt.treasuredata.com/sh/install-ubuntu-xenial-td-agent2.sh | sh && \
-    sed -i -e "s/USER=td-agent/USER=root/" -e "s/GROUP=td-agent/GROUP=root/" /etc/init.d/td-agent && \
-    td-agent-gem install --no-document fluent-plugin-record-reformer -v 0.8.2 && \
-    td-agent-gem install --no-document fluent-plugin-systemd -v 0.0.5 && \
-    td-agent-gem install --no-document fluent-plugin-google-cloud -v 0.5.2 && \
-    td-agent-gem install --no-document fluent-plugin-detect-exceptions -v 0.0.4 && \
-    td-agent-gem install --no-document fluent-plugin-influxdb && \
-    td-agent-gem install --no-document fluent-plugin-kubernetes_metadata_filter && \
-    td-agent-gem install --no-document fluent-plugin-flatten-hash && \
-    # Remove build tools
-    apt-get remove -y -qq gcc make && \
-    apt-get autoremove -y -qq && \
-    apt-get clean -qq && \
-    # Remove unnecessary files
-    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* \
-        /opt/td-agent/embedded/share/doc \
-        /opt/td-agent/embedded/share/gtk-doc \
-        /opt/td-agent/embedded/lib/postgresql \
-        /opt/td-agent/embedded/bin/postgres \
-        /opt/td-agent/embedded/share/postgresql \
-        /etc/td-agent/td-agent.conf
-
-# Copy the Fluentd configuration file for logging Docker container logs.
-COPY fluent.conf /etc/td-agent/td-agent.conf
-
-# Copy the entrypoint for the container
-COPY run.sh /run.sh
-
-# Start Fluentd to pick up our config that watches Docker container logs.
-CMD /run.sh $FLUENTD_ARGS
+FROM gcr.io/google-containers/fluentd-gcp:2.0.18
+
+# install additional plugins.
+RUN BUILD_DEPS="make gcc g++ libc6-dev ruby-dev libffi-dev" \
+    && clean-install $BUILD_DEPS \
+    && echo "gem 'fluent-plugin-influxdb', '~>1.1.0'" >> Gemfile \
+    && echo "gem 'fluent-plugin-kubernetes_metadata_filter', '~>1.2.1'" >> Gemfile \
+    && echo "gem 'fluent-plugin-flatten-hash', '~>0.4.1'" >> Gemfile \
+    && gem install --file Gemfile \
+    && apt-get purge -y --auto-remove \
+                     -o APT::AutoRemove::RecommendsImportant=false \
+                     $BUILD_DEPS \
+    && rm -rf /tmp/* \
+              /var/lib/apt/lists/* \
+              /usr/lib/ruby/gems/*/cache/*.gem \
+              /var/log/* \
+              /var/tmp/*
+
+COPY fluent.conf /etc/fluent/fluent.conf
```

Comment:
Now, the mount path of config file of logger is at /etc/td-agent/
https://github.com/fission/fission/blob/1dbb4f4fb73ed1a3be97f87b059c8a5e1ebc9536/charts/fission-all/templates/fluentd.yaml#L63
you need to change the path in both charts as well.

Reply:
Sorry, this is an important PR and affect the CI build, I addressed the issue to make CI back to live again.